source/served/extension.d

```diff
@@ -991,9 +1042,43 @@ void onDidSaveDocument(DidSaveTextDocumentParams params)
 	}
 }
 
+@protocolNotification("workspace/didChangeWatchedFiles")
+void onDidChangeWatchedFiles(DidChangeWatchedFilesParams params)
```

Comment:
this function name is too generic, it should be called something with CCDB as it only handles that.

Additionally this function and `discoverCcdb` should be moved into a separate ccdb module to avoid cluttering extension with new features that could be nicely grouped together. When you create a separate module, you need to `static import` it in extension.d (see the top of the file) and add it to the list of members at the bottom of the file